src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -44,7 +47,9 @@ internal AssemblySymbol CorLibrary
         {
             get
             {
+#nullable disable // Can '_corLibrary' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
It appears that after initialization, the answer is no. It can take a bit to be initialized sometimes, but even `MissingCorLibrarySymbol` uses a sentinel. Appropriate thing is probably to assert it's never null and return.

Reply:
I'm planning to leave this for a follow-up